apps/coscrad-frontend/src/store/slices/resources/shared/notes-for-resource/self-notes-panel.presenter.tsx

```diff
@@ -0,0 +1,26 @@
+import { ResourceCompositeIdentifier } from '@coscrad/api-interfaces';
+import { SelfConnectionNote } from '../../../notes/hooks/use-loadable-self-notes-for-resource';
+import { SelfNotePresenter } from './self-note.presenter';
+
+interface SelfNotesPanelProps {
+    compositeIdentifier: ResourceCompositeIdentifier;
+    notes: SelfConnectionNote[];
+}
+
+export const SelfNotesPanelPresenter = ({
+    compositeIdentifier: { type: resourceType, id },
+    notes,
+}: SelfNotesPanelProps): JSX.Element => (
+    // TODO remove magic string
+    <div data-testid={'selfNotesPanel'}>
+        <h2>
+            {/* TODO Use standard formatter */}
```

Comment:
Should we use the resource info from Redux? Or should we share the formatters statically with the back-end through a lib?

Reply:
[Here](https://www.pivotaltracker.com/story/show/183867993) is a story for that.